postgresql/postgresql_plugin.cc

```diff
@@ -0,0 +1,832 @@
+/* postgresql_plugin.cc
+   Mathieu Marquis Bolduc, 16 July 2016
+   Copyright (c) 2015 Datacratic Inc.  All rights reserved.
+*/
+
+#include "mldb/core/plugin.h"
+#include "mldb/core/procedure.h"
+#include "mldb/core/function.h"
+#include "mldb/core/dataset.h"
+#include "mldb/soa/credentials/credentials.h"
+#include "mldb/types/structure_description.h"
+#include "mldb/types/any_impl.h"
+
+#include <postgresql/libpq-fe.h>
```

Comment:
We opted to use libpq from the system packages instead on bundling it.

On Aug 10, 2016 9:46 PM, "Guy Dumais" <notifications@github.com> wrote:

> In postgresql/postgresql_plugin.cc
> <https://github.com/mldbai/mldb/pull/601#discussion_r74360514>:
>
> > @@ -0,0 +1,832 @@
> > +/* postgresql_plugin.cc
> > +   Mathieu Marquis Bolduc, 16 July 2016
> > +   Copyright (c) 2015 Datacratic Inc.  All rights reserved.
> > +*/
> > +
> > +#include "mldb/core/plugin.h"
> > +#include "mldb/core/procedure.h"
> > +#include "mldb/core/function.h"
> > +#include "mldb/core/dataset.h"
> > +#include "mldb/soa/credentials/credentials.h"
> > +#include "mldb/types/structure_description.h"
> > +#include "mldb/types/any_impl.h"
> > +
> > +#include <postgresql/libpq-fe.h>
>
> Any reason why this is imported as a system header?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mldbai/mldb/pull/601/files/b1eb743292c207093362480006e6e5e443690f37#r74360514>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACQXTbWtrEFB28ptz1cWAnO9wgnLGpHJks5qen7fgaJpZM4JhiXj>
> .
>